### DISCUSSION TITLE
光映广场的台阶还能帮人开胯吗

### DISCUSSION THREAD

**Discussion by hapildk** [share121/inter-knot#5263] [灌水]
无聊的时候乱逛发现的
![Uploading IMG_20240921_120713.jpg…]()

**Comment by cpu644** [comment]
你的图片没有上传成功
![image](https://github.com/user-attachments/assets/edcb704b-667c-4d98-b76d-20cb518f468d)
你需要让图片这么显示才算发送成功
![image](https://github.com/user-attachments/assets/13809771-ede5-4af5-b7b7-765fba1ef505)
